platform-hub-api/app/controllers/app_settings_controller.rb

```diff
@@ -1,5 +1,10 @@
 class AppSettingsController < ApiJsonController
 
+  PUBLIC_FIELDS = [
+    'platformName',
```

Comment:
`platform_name` ?

Reply:
Unfortunately we've been using `platformName` so far. This was because the web UI actually controls what goes into the app settings – the API has no schema of what can be in here (at the moment). So we've ended up with camel casing here.